cmd/test/main.go

```diff
@@ -292,6 +311,7 @@ func GenerateTestCommandData(testCommandFlags *TestCommandFlags, localConfigCont
 		ClientId:              localConfigContent.ClientId,
 		RegistrationURL:       evaluationPrerunDataResp.RegistrationURL,
 		PromptRegistrationURL: evaluationPrerunDataResp.PromptRegistrationURL,
+		SkipSchemaValidation:  testCommandFlags.SkipValidation == "schema",
```

Comment:
what will happen in the future if we will have an array of possible validations?

Reply:
In this case we will change the flags internally, but it won't damage the user experience since the real flag will stay `--skip-validation=schema`